fearless/Common/Services/ExtrinsicService/ExtrinsicService.swift

```diff
@@ -259,4 +424,41 @@ extension ExtrinsicService: ExtrinsicServiceProtocol {
 
         operationManager.enqueue(operations: wrapper.allOperations, in: .transient)
     }
+
+    func submit(
+        _ closure: @escaping ExtrinsicBuilderIndexedClosure,
+        signer: SigningWrapperProtocol,
+        runningIn queue: DispatchQueue,
+        numberOfExtrinsics: Int,
+        completion completionClosure: @escaping ExtrinsicSubmitIndexedClosure
+    ) {
+        let wrapper = operationFactory.submit(closure, signer: signer, numberOfExtrinsics: numberOfExtrinsics)
+
+        wrapper.targetOperation.completionBlock = {
+            queue.async {
+                if let operationResult = wrapper.targetOperation.result {
```

Comment:
Please, use extractNonCancellable result to avoid duplicated code for error handling